components/sections/home/ReadWhitepaper.tsx

```diff
@@ -51,23 +51,25 @@ function ReadWhitepaper() {
                 fontWeight="medium"
                 color="brand.white"
               >
-                Shardeum is a smart contract platform that scales{" "}
+                {/* Shardeum is a smart contract platform that scales{" "}
                 <Text as="span" color="brand.orange">
                   linearly
                 </Text>{" "}
                 and infinitely through{" "}
                 <Text as="span" color="brand.orange">
                   dynamic state sharding{" "}
-                </Text>{" "}
+                </Text>{" "} */}
+                The first linearly scalable smart contract platform that increases transactions per
```

Comment:
Highlighted text is missing? Was it intentional?
![image](https://user-images.githubusercontent.com/36589645/167419828-291aaadd-43a7-4618-b6c3-8ad82fa47957.png)

Main Website

![image](https://user-images.githubusercontent.com/36589645/167419896-66f40ffb-8145-45f6-9202-84b147878dc7.png)

Reply:
Fixed this